indicators/base64-url-encoded-body.yml

```diff
@@ -0,0 +1,19 @@
+title: Base64 & URL-encoded document body
+description: |
+  To evade static analysis, the document body can be wrapped in several
+  JavaScript functions such as `decodeURIComponent` and `atob` in order
+  to evade analysis.
+  This helps defeat simple scanners which don't evaluate JavaScript.
+  
```

Comment:
```suggestion
related:
  - hex-encoded-body
```

Reply:
I'll keep this noted for future reference, thanks!